src/main/webapp/app/overview/courses-routing.module.ts

```diff
@@ -92,15 +79,6 @@ const routes: Routes = [
         },
         canActivate: [UserRouteAccessService],
     },
-    {
-        path: 'courses/:courseId/exercises/:exerciseId/teams/:teamId',
-        component: TeamComponent,
-        data: {
-            authorities: [Authority.USER],
-            pageTitle: 'artemisApp.team.detail.title',
-        },
-        canActivate: [UserRouteAccessService],
-    },
```

Comment:
By removing those lines, a student is not able to click the team button anymore:
![image](https://user-images.githubusercontent.com/65229601/142653813-25e6405e-a4c6-4309-af18-1878377b991e.png)
Therefor, the student is not able to navigate into this page
![image](https://user-images.githubusercontent.com/65229601/142654036-a2336336-eeec-4098-893c-c2ada5f1bebc.png)

Reply:
I think it was resolved by 632306c2988571ce5b9cd6025540a0e40e05c54d